plugins/inputs/kafka_consumer/kafka_consumer.go

```diff
@@ -132,7 +132,7 @@ func (k *KafkaConsumer) Init() error {
 	}
 
 	if err := k.SetConfig(cfg, k.Log); err != nil {
-		return fmt.Errorf("SetConfig: %w", err)
+		return fmt.Errorf("failed to SetConfig: %w", err)
```

Comment:
I'm a fan of
```suggestion
		return fmt.Errorf("setting config failed: %w", err)
```
to not bother users with strange function names but rather with what was the plan. :-)

Reply:
Done.